src/AutoSysimages.jl

```diff
@@ -411,7 +411,7 @@ Feel free to submit a PR."""
     txt = if Sys.iswindows()
 """@echo off
 set JULIA=$julia_bin
-for /f "tokens=1-4" %%i in ('%JULIA% -L $julia_args_file') do set A=%%i %%j %%k %%l
+for /f "tokens=1-4" %%i in ('%JULIA% -L $julia_args_file %*') do set A=%%i %%j %%k %%l
```

Comment:
Why use `%*` two times in the script ?

Reply:
Because you need to pass `--project` argument in both cases to detect the active project correctly.